index.html

```diff
@@ -3674,11 +3708,15 @@ <h3>HTML Attribute State and Property Mappings</h3>
               </td>
               <td class="ax">
                 <span class="type">Role:</span>
-                <a href="#el-textarea">AXtextArea</a>
+                <a href="#el-textarea">AXTextArea</a>
```

Comment:
Keeping unrelated changes out of the diff... See next comment thread (applied to line 3678 for some reason) 

👀 Watching you two!
```suggestion
                <a href="#el-textarea">AXtextArea</a>
```

Reply:
Referenced "[next comment](https://github.com/w3c/html-aam/pull/454#discussion_r1091230177)" is hidden in an outdated thread, so repeating here. 

> We shouldn't screw up the blame traces, and we don't need merge conflicts, especially since we'll probably remove that entire line with #455